### ISSUE TITLE
Board Copy Bug

### ISSUE THREAD

**Issue by xorander00** [RestyaPlatform/board#985]
When copying a board, including cards, it doesn't copy the state of the checklists within cards to the new board.  Not sure if this is intentional, but I just noticed it.

**Comment by rrjanbiah** [comment]
@xorander00 Thanks for your report!

**Comment by lmarikannan** [comment]
@xorander00  Yes. We have done the functionality like that. Because in new board, user have to complete those checklist for that copied board.

**Comment by lmarikannan** [comment]
@xorander00 Also if you have any suggestion for this functionality, please feel free to reopen.

**Comment by xorander00** [comment]
No problem.  That's actually fine.  If anything, then maybe an additional option to select which sub-items to copy during a board copy.  I wouldn't prioritize it though unless it's an issue for others.

I'd probably document it though somewhere.  Probably have a question mark icon next to the board copy operation that, when hovered, points of which parts of the board are specifically copied.  I lost a little data this way because I didn't realize that it didn't copy the state of the checklists.  I realized it when I went back to it later and saw that some items I knew I had completed were not checked.

Making it clear to the user would probably save some headaches.
